orchestration/tests/test_stale_task_prevention.py

```
@@ -0,0 +1,413 @@
+- Agents reused old prompt files containing different tasks
+- User requested "server modification", agents executed "PR comment responses"
+- Root cause: Missing cleanup lifecycle for task prompt files
+
+Test Goals:
+1. Verify prompt files are cleaned up between orchestration runs
+2. Ensure fresh task prompts are generated for each new request
+3. Test multi-run scenarios to catch stale state reuse
+4. Verify task execution matches user intent, not cached prompts
+"""
+
+import glob
```

Ruff (0.12.2)

`glob` imported but unused

Remove unused import: `glob`

(F401)